### ISSUE TITLE
Compatiblity with Gradle 8.0

### ISSUE THREAD

**Issue by agacemi** [adjust/flutter_sdk#107]
We try to upgrade to Android Gradle Plugin 8.0. Build fails with the fellowing error due to breaking change in AGP: namespace is mandatory. https://developer.android.com/build/releases/gradle-plugin#namespace-dsl

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':adjust_sdk'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Please specify a namespace in the module's build.gradle file like so:

     android {
         namespace 'com.example.namespace'
     }

     If the package attribute is specified in the source AndroidManifest.xml, it can be migrated automatically to the namespace value in the build.gradle file using the AGP Upgrade Assistant; please refer to https://developer.android.com/studio/build/agp-upgrade-assistant for more information.
```

**Comment by dihsarjr** [comment]
**I am also facing the same issue.**


FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':adjust_sdk'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Please specify a namespace in the module's build.gradle file like so:
     
     android {
         namespace 'com.example.namespace'
     }
     
     If the package attribute is specified in the source AndroidManifest.xml, it can be migrated automatically to the namespace value in the build.gradle file using the AGP Upgrade Assistant; please refer to https://developer.android.com/studio/build/agp-upgrade-assistant for more information.

* Try:
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

**Comment by daiki1003** [comment]
anything update?

**Comment by agacemi** [comment]
Any update ? We can't upgrade to Gradle 8.0 due to this error

**Comment by daiki1003** [comment]
Do we have to remove adjust sdk...?

**Comment by uerceg** [comment]
Hey guys,

Thank you for reporting this and sorry for the delay. We'll take a look and keep you posted.

**Comment by daiki1003** [comment]
Thaks!
I know this is a lot of work, but thank you in advance!

**Comment by daiki1003** [comment]
Anything update？

**Comment by uerceg** [comment]
Hey guys, sorry for a bit of delay on this one. We have just released [v4.34.0](https://github.com/adjust/flutter_sdk/releases/tag/v4.34.0) which should hopefully make the plugin compatible with Gradle 8.0++. If you can please give it a shot and let us know if it worked, that would be great. Cheers.
